### ISSUE TITLE
Updating extracted servers

### ISSUE THREAD

**Issue by cloggier** [hrsh7th/vscode-langservers-extracted#2]
Hi @hrsh7th, great thanks for that package! I was not aware how out of date -bin versions were.

I would like to help you with maintaining this repository. If you could talk me through procedure of updating servers in this repository I can be sending you PRs once upstream releases new versions.

Currently we have:
```
"vscode-css-languageservice": "^5.0.2",
"vscode-html-languageservice": "^4.0.0",
"vscode-json-languageservice": "^4.0.0",
"vscode-languageserver": "^7.0.0",
```

But there already are: `css: 5.1.0`, `json: 4.0.2` and `html: 4.0.1` - neither very far behind, but would be good to keep up.

**Comment by hrsh7th** [comment]
I'm updating this randomly when I notice whatever VSCode's extensions were updated.

Currently, the json/html/css servers seem to have a new feature that is able to drop MDN references.

OK. I will update it.

By the way, This repository can't update by other persons because it just automatically downloads and publish it as npm.

**Comment by hrsh7th** [comment]
I've published v2.3.1

**Comment by cloggier** [comment]
Awesome, thank you!
